addons/account/tests/test_account_invoice_report.py

```diff
@@ -212,3 +212,73 @@ def test_avg_price_calculation(self):
             ['price_average:avg'],
         )
         self.assertEqual(round(report[0]['price_average:avg'], 2), 4.71)
+
+    def test_avg_price_group_by_month(self):
+        """
+        Check that the average is correctly calculated based on the total price and quantity
+        with multiple multipleinvoice and group by month:
+            Invoice 1:
+                2 lines:
+                    - 10 units * 10$
+                    -  5 units *  5$
+                Total quantity: 15
+                Total price: 125$
+                Average: 125 / 15 = 8.33
+            Invoice 2:
+                1 line:
+                    - 0 units * 5$
+                Total quantity: 0
+                Total price: 0$
+                Average: 0.00
+        """
+        self.env['account.move'].search([]).unlink()
+        invoices = self.env["account.move"].create([
+            {
+                'move_type': 'out_invoice',
+                'partner_id': self.partner_a.id,
+                'invoice_date': fields.Date.from_string('2025-01-01'),
+                'currency_id': self.env.company.currency_id.id,
+                'invoice_line_ids': [
+                    (0, None, {
```

Comment:
```suggestion
                    Command.create({
```

You should import and use `Command` instead.